app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -88,51 +81,55 @@ function fetchNotificationsCount() {
   }
 }
 
+const onClick = event => {
```

Comment:
Can you change the name of the function with something more descriptive?

Reply:
Sure